packages/cqrs/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 10.0.1
+
+- **Breaking:** Rename `forbiddenAccess` to `authorization` in error enums.
```

Comment:
I'll let @Albert221 decide what to do here. Clearly its not a patch release as it is breaking. But I understand the sentiment that this is a hotfix over v10. I don't think pub has any process of yanking releases.

Reply:
I'll mark 10.0.0 as retracted in Pub. Please mark 10.0.0 as retracted also in changelog @jtarkowski27

Reply:
Good to know there is some yanking support in pub